### ISSUE TITLE
I think these dimensions are off in the comment

### ISSUE THREAD

**Issue by tslevi-unbounce** [unbounce/pytorch-tree-lstm#8]
https://github.com/unbounce/pytorch-tree-lstm/blob/66f29a44e98c7332661b57d22501107bcb193f90/treelstm/tree_lstm.py#L92

**Comment by shravansingh123** [comment]
Yes I agree. It should be e x out_features . Also              c[node_mask, :] = i * u
will not work because c dimension is [batch_size,out_features] and batch_size and node_mask doesn't have to be same. Batch_size is one in case of single tree. Did you stumble upon the same problem?

**Comment by shravansingh123** [comment]
https://github.com/unbounce/pytorch-tree-lstm/blob/66f29a44e98c7332661b57d22501107bcb193f90/treelstm/tree_lstm.py#L116
